src/test.ts

```diff
@@ -0,0 +1,4 @@
+///<reference path="./deno/stable/lib.deno.d.ts" />
+
+/** Reference to the array that `Deno.test` calls insert their definition into. */
+export const testDefinitions: Deno.TestDefinition[] = [];
```

Comment:
I think this should be in `src/deno/internal`; what do you think of the public testing API additionally exposing `readTests`?

All we need is:

```TS
export const readTests = () => testDefinitions.splice(0);
```

`package.json` should likely have something to the effect of:

```JSON
"exports": {
	"./test-internals": "./dist/deno/internal/test.js"
}
```